jobs/aws/eks-distro/aws-cloud-controller-manager-1-26-presubmits.yaml

```diff
@@ -51,9 +51,9 @@ presubmits:
           &&
           build/lib/local_registry_check.sh
           &&
-          if make check-for-supported-release-branch -C $PROJECT_PATH; then make build clean-go-cache clean -C $RELEASE_PROJECT_PATH IMAGE_OUTPUT_TYPE=image IMAGE_OUTPUT=push=true; fi
```

Comment:
Do we not need that check anymore?

Reply:
I wondered about this too, but it's what we did in 1.26. I remember asking Jackson about this and he said it was something to do with caching and only if for the most recent release. So like in 1.26, it made these same changes to the 1.25 files https://github.com/aws/eks-distro-prow-jobs/pull/464/files